rafthttp/msgapp.go

```diff
@@ -35,11 +35,15 @@ type msgAppEncoder struct {
 }
 
 func (enc *msgAppEncoder) encode(m raftpb.Message) error {
+	start := time.Now()
```

Comment:
decoding is special. i do not want to count wait time. encoding 8bytes is nothing. let's not worry about it. it does not have to symmetric.